### DISCUSSION TITLE
Manual installation (without HACS)

### DISCUSSION THREAD

**Discussion by 8arcly** [dbuezas/lovelace-plotly-graph-card#161] [Q&A] [answered]
Hi, are there some instructions to install Plotly graph card manually (without HACS)?

**Comment by dbuezas** [comment, marked as answer]
There are now! :) 

See here: https://github.com/dbuezas/lovelace-plotly-graph-card#manually

**Reply by 8arcly** [reply]
Thanks! Works perfectly :)
